server/cluster/cluster.go

```diff
@@ -1095,7 +1094,11 @@ func (c *RaftCluster) buryStore(storeID uint64) error {
 	}
 
 	if store.IsUp() {
-		return errs.ErrStoreIsUp.FastGenByArgs()
+		if !forceBury {
+			return errs.ErrStoreIsUp.FastGenByArgs()
+		} else if !store.IsDisconnected() {
```

Comment:
I think we don't need this condition.

Reply:
This is also part of the design, we are encouraging our customers only use this feature only when the store is broken and unrecoverable, which means, if a store is up and connected, we disallow the operation.